tests/javascript/newsletter_editor/blocks/automatedLatestContentLayout.spec.js

```diff
@@ -474,6 +481,12 @@ describe('Automated latest content layout', function () {
         expect(model.get('featuredImagePosition')).to.equal(newValue);
       });
 
+      it('changes the model if featured image position changes', function () {
+        var newValue = 'aboveExcerpt';
```

Comment:
Seems like you're changing from default value `aboveExcerpt` to nevValue `aboveExcerpt`, is that so? In that case it would be a nicer test to try the other value.

Reply:
Why do you think that? I can see the default is `abovePost`: https://github.com/mailpoet/mailpoet/pull/1852/files#diff-fd1c6f0c7468a26e792008e610774b58R69

And even if I try it, it seems to be the case: https://d.pr/i/JsxQ8z

Reply:
I was confused by this line https://github.com/mailpoet/mailpoet/pull/1852/commits/eb0c27144bf3fb90064c9851b824056bf2dfed81#diff-8adc3011f2281cb263b761ca63086374R192, but actually that is a test as well.